src/main.py

```diff
@@ -300,3 +331,7 @@ def detect():
             return 500
         finally:
             tempFile.close()
+
+@app.route('/debug-sentry')
```

Comment:
Should we remove this path?

Reply:
I Do not think it's a problem that it's there, let's leave for now. I can remove it later when it's tested